src/delay.ml

```diff
@@ -165,7 +165,8 @@ let pp_explain ppf d =
 
 (** Parsing *)
 
-module Parsing = struct
+
+module Iso8601 = struct
```

Comment:
I had to keep this piece of code for testing. Does `daypack` provides a restricted parser for ISO8601 ?

Reply:
Not atm, no - probably should have one shouldn't it...(opening an issue)